deployment/helm/readme.md

```diff
@@ -0,0 +1,30 @@
+Helm Chart Deployment for Greenfield Validator
+
+## To deploy a Greenfield Validator node:
+1. `helm repo add bnb-chain https://chart.bnbchain.world/`
+2. `helm repo update`
+3. `helm install greenfield-validator bnb-chain/gnfd-validator`
+
+## To note:
+Please ensure you have VMServiceScrape CRD installed in your cluster, else, the helm chart cannot install and deploy the resources below into your cluster.
```

Comment:
We should put this before the upper deployment commands as Dependence?

There should be no : in the header.